### PR TITLE
EPMRPP-62755 Fixed the position of the cross sign

### PR DESCRIPTION
**Pull request by katrin-kot**

### CODE REVIEW

File: app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss [hunk 1]
```diff
@@ -54,8 +54,8 @@
 
 .remove-issue-fields-button {
   position: absolute;
-  right: -4px;
-  top: 25px;
+  right: 5px;
+  top: 12px;
```
Comment (chivekrodis) [inline app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss:58]: seems like better align cross sign in the middle the `link to issue` field
please, discuss with task reporter
image

Comment (katrin-kot) [reply app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss]: Fixed the position to the level of the upper border

Comment (AmsterGet) [reply app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss]: I thnk it was a bug that the crosses where placed not near the inputs.
The best choice here is (see the screenshot below)

image

It is an initial placement for this crosses discussed with designer, so please move them according to the picture.

File: app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss [hunk 2]
```diff
@@ -54,8 +54,8 @@
 
 .remove-issue-fields-button {
   position: absolute;
-  right: -4px;
-  top: 25px;
+  right: 40%;
```
Comment (AmsterGet) [inline app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss:57]: Not the crosses locates in diffrent places in show less/more view.

Show less:

image

Show more:

image

May be better to use static indentation for the crosses.
Just add `display: inline-block` to the `issue-fields-list-item` and `right: -50px` to the `remove-issue-fields-button`.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**